src/liveEditor/utils/handleIndividualFields.ts

```diff
@@ -138,9 +140,13 @@ export async function handleIndividualFields(
             const elementComputedDisplay =
                 window.getComputedStyle(actualEditableField).display;
 
-            const textContent =
+            let textContent =
                 (editableElement as HTMLElement).innerText || editableElement.textContent || "";
 
+            if(fieldType === FieldDataType.MULTILINE) {
```

Comment:
I think we will have to account for psuedo editable multiline as well here.

Reply:
sure, on it